code/modules/mob/living/carbon/superior_animal/robots/subtype/greyson/types/turrets.dm

```diff
@@ -78,6 +78,93 @@
 
 
 /mob/living/carbon/superior_animal/robot/greyson/stalker/dual/New()
-	..()
-	if(prob(1))
+	. = ..()
+	if(prob(1) && (!drop2))
 		drop2 = /obj/item/gun/projectile/automatic/scaffold
+
+
+/obj/item/stalker_fuel_rod
+	name = "plasma fuel rod"
+	desc = "A greyson-alloyed fuel rod, filled to the brim with plasma."
+
+	origin_tech = list(TECH_GREYSON = 5, TECH_PLASMA = 3, TECH_ENGINEERING = 2)
+	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASMA = 1, MATERIAL_PLASMAGLASS = 1, MATERIAL_DIAMOND = 2)
+
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "fuel_rod"
+
+	preloaded_reagents = list("plasma" = 30)
+
+/obj/item/stalker_fuel_rod/depleted
+	name = "spent fuel rod"
+	desc = "A empty greyson-alloyed fuel rod, previously having contained some combustable substance. Presumably."
+
+	icon_state = "fuel_rod_depleted"
+
+	preloaded_reagents = null
+
+// mini-boss enemy that demands attention or else they will nuke someone. VERY POWERFUL
+/mob/living/carbon/superior_animal/robot/greyson/stalker/dual/plasma_cannon
+	name = "\"Iron Lock Security\" Assault Stalker Mk2"
```

Comment:
```suggestion
	name = "\"Iron Lock Security\" Assault Stalker MkIV"
```

Reply:
breaks the naming convention of stalkers, we dont use roman numerals